modules/shared/src/test/scala/org/tessellation/schema/generators.scala

```diff
@@ -127,4 +129,35 @@ object generators {
 
   val signedTransactionGen: Gen[Signed[DAGTransaction]] = signedOf(transactionGen)
 
+  val snapshotOrdinalGen: Gen[SnapshotOrdinal] =
+    Arbitrary.arbitrary[NonNegLong].map(SnapshotOrdinal(_))
+
+  val heightGen: Gen[Height] =
+    Arbitrary.arbitrary[NonNegLong].map(Height(_))
+
+  val subHeightGen: Gen[SubHeight] =
+    Arbitrary.arbitrary[NonNegLong].map(SubHeight(_))
+
+  val globalSnapshotGen =
```

Comment:
this generator won't pass snapshot validation. I'd rather create a proper snapshot instead of using arbitrary gen

Reply:
Is this function (taken from SnapshotProcessorSuite) creating a valid snapshot?
```
  def generateSnapshot(peerId: PeerId): GlobalSnapshot =
    GlobalSnapshot(
      snapshotOrdinal10,
      snapshotHeight6,
      snapshotSubHeight0,
      lastSnapshotHash,
      SortedSet.empty,
      SortedMap.empty,
      SortedSet.empty,
      EpochProgress.MinValue,
      NonEmptyList.one(peerId),
      GlobalSnapshotInfo(SortedMap.empty, SortedMap.empty, SortedMap.empty),
      SnapshotTips(SortedSet.empty, SortedSet.empty)
    )
```

Reply:
yes, I think so. `SnapshotProcessorSuite` uses the full validation

Reply:
I thought I based my generator on that function. I'll check again this morning.

Reply:
it has predefined/hardcoded variables I think. the only concern I have is about the `lastSnapshotHash` and I think that `Arbitrary[Hash]` produces wrong hashes

Reply:
globalSnapshotGen removed